components/web/squid/Makefile

```diff
@@ -30,18 +30,16 @@ USE_OPENSSL11=		yes
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=         squid
-COMPONENT_VERSION=      5.5
-COMPONENT_REVISION=		1
+COMPONENT_VERSION=      5.6
+COMPONENT_REVISION=	1
 COMPONENT_SUMMARY=      Squid Web Proxy Cache
```

Comment:
Remove COMPONENT_REVISION here, it needs to be reset after COMPONENT_VERSION update.

Reply:
Sorry forgot about that. Removed now.